### PR TITLE
#CP-38 feat: update profile

### PR DESCRIPTION
**Pull request by Yvad60**
#### What the PR does
it enables users (operator or driver) to update information on their profiles so that others could see updated information about them.

#### Description of Tasks to be completed

- [x] The operator and driver should be able to update information about their own profile.
- [x] The operator and driver profile should be able accessible publicly.
- [x] Appropriate error messages are displayed on unsuccessful attempts.

#### How this should be manually tested
Clone the project repository linked [here](https://github.com/atlp-rwanda/phantom-be-codebandits)
checkout to `ft-update-profile-cp-38` branch
in postman try to create a new driver or operator using some info
try to edit the info of the created user
users should not be able to edit their email, password, company, or role test if that works too

#### Relevant Trello Card
[#CP-38](https://trello.com/c/NKv7l2Kq/38-update-profile)

### CODE REVIEW

File: src/operatorApp/models.js [hunk 1]
```diff
@@ -3,8 +3,6 @@ import { BaseEntity, EntitySchema } from 'typeorm';
 export class Operator extends BaseEntity {
 	id;
 
-	email;
```
Comment (feyton) [inline src/operatorApp/models.js:6]: This was out of place

**Review by renepromesse — CHANGES_REQUESTED** [review]
Good work @Yvad60 ,
please improve these failing test

**Review by feyton — DISMISSED** [review]
### Approval
This has been a great feat @Yvad60 . This implementation is really great and since it can even be scaled to be used in other program components. This is great work

File: locales/en.json [hunk 2]
```diff
@@ -42,11 +42,12 @@
 	"driver_deleted": "driver deleted successfully",
 	"zero_users": "No users in the database",
 	"address_required": "Adrress is required",
-	"operator_deleted": "operator_deleted",
+	"operator_deleted": "Operator deleted successfully",
 	"driver_updated": "Driver updated successfully",
 	"national_ID_exist": "User with provided national ID exists",
 	"license_ID_exist": "User with provided license exists",
 	"not_Found": "not_Found",
```
Comment (Abdulkeza) [inline locales/en.json]: Good job Mr. Yvad, if you are working on feedback from others, remember to change this, looks like the key and value are the same... on line 45 and 49

**Review by Abdulkeza — CHANGES_REQUESTED** [review]

File: locales/en.json (hunk 2 above)
Comment (Yvad60) [reply locales/en.json]: yeahhh! thanks

**Review by samyshyaka — DISMISSED** [review]
Nice work @Yvad60, This is a good implementation of the feature. I think the PR is ready to be merged

**Review by Abdulkeza — APPROVED** [review]
Great job Mr ivad. it work fine

**Review by samyshyaka — APPROVED** [review]

**Review by renepromesse — APPROVED** [review]

**Pull request merged**